openmc/lattice.py

```diff
@@ -378,6 +366,51 @@ def get_all_universes(self):
 
         return all_universes
 
+    def get_universe(self, idx):
+        """Return universe corresponding to a lattice element index
+
+        Parameters
+        ----------
+        idx : Iterable of int
```

Comment:
This should mention the support for both 2D and 3D lattices. Also, correct me if I'm wrong, but the indices are referenced in the order of z, y, and x in lines 385 and 387, right?